pkg/test/resourcefixture/contexts/kms_context.go

```diff
@@ -29,4 +29,15 @@ func init() {
 		//   has value DESTROY_SCHEDULED in field crypto_key_version.state., failedPrecondition
 		SkipDriftDetection: true,
 	}
+	resourceContextMap["kmsautokeyconfig"] = ResourceContext{
+		ResourceKind: "KMSAutokeyConfig",
+		// TestCreateNoChangeUpdateDelete/basic-kmscryptokey: dynamic_controller_integration_test.go:149: value
```

Comment:
Does autokeyconfig support labels?

Reply:
in the apis we dont have any field to take `labels` as input, so assuming it doesnt support labels